test/views/UnloadingGuidanceViewSpec.scala

```diff
@@ -16,33 +16,106 @@
 
 package views
 
+import models.NormalMode
 import play.twirl.api.HtmlFormat
+import viewModels.UnloadingGuidanceViewModel
 import views.behaviours.ViewBehaviours
 import views.html.UnloadingGuidanceView
 
 class UnloadingGuidanceViewSpec extends ViewBehaviours {
 
+  private val viewModel = new UnloadingGuidanceViewModel(false, Some(true))
+
+  override def view: HtmlFormat.Appendable =
+    injector.instanceOf[UnloadingGuidanceView].apply(mrn, arrivalId, messageId, NormalMode, viewModel)(fakeRequest, messages)
+
+  override val prefix: String = "unloadingGuidance.notNewAuth"
+
+  behave like pageWithTitle(false, true)
+
+  behave like pageWithBackLink()
+
+  behave like pageWithCaption(s"This notification is MRN: ${mrn.toString}")
+
+  behave like pageWithHeading(false, true)
+
+  behave like pageWithContent(
+    "p",
+    s"When unloading, check that the goods match the unloading permission for Movement Reference Number (MRN) $mrn. Take note of any discrepancies as you will need to include them in your unloading remarks."
+  )
+
+  behave like pageWithLink(
+    id = "download",
+    expectedText = "Download the Unloading Permission PDF",
+    expectedHref = s"http://localhost:9485/manage-transit-movements/${arrivalId.value}/unloading-permission-document/$messageId"
+  )
+
+  behave like pageWithSubmitButton("Continue")
+}
+
+class UnloadingGuidanceViewSpecNewAuthGoodsTooLargeSpec extends ViewBehaviours {
```

Comment:
There's be no need for these separate specs if we define our view model params to have no dependency on the booleans. The apply method should depend on the booleans instead.